packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.Props.ts

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { IStyle } from '../../Styling';
+import { IPersonaProps } from 'office-ui-fabric-react/lib/Persona';
+
+// Please keep alphabetized
+export interface IActivityItemProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * Text or elements in this array will be shown in order as the description of the activity that occurred.
+   */
+  activityDescription: Array<string | JSX.Element>;
+
+  /**
+   * Text or elements of a comment that will be shown in order under the activity description.
+   */
+  commentElements?: Array<string | JSX.Element>;
+
+  /**
+   * Used to render the main icon. Pass in an array of Persona props to automatically render a Persona group, or pass in a JSX element to render your own content, such as an Icon, here.
+   */
+  iconContents: Array<IPersonaProps> | JSX.Element;
+
+  /**
+   * Indicated if the compact styling should be used.
+   */
+  isCompact?: boolean;
+
+  /**
+   * A renderer adds a time stamp. If not included, timeString is shown as plain text below the activity.
+   */
+  onRenderTimeStamp?: (props?: IActivityItemProps) => JSX.Element;
```

Comment:
Change this to an IRenderFunction, along with the other props. This allows a dev to replace the timestamp, or just amend it. 

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/BESTPRACTICES.md#custom-render-methods